amanuensis/resources/userdatamodel/userdatamodel_project.py

```diff
@@ -67,7 +67,7 @@ def get_project_by_user(current_session, logged_user_id, logged_user_email):
     )
 
 
-def get_project_by_id(current_session, logged_user_id, project_id):
```

Comment:
This piece breaks some other functions, those would need to be updated or we could leave this in and remove it in a future update